tests/test_pipelines_conversational.py

```diff
@@ -295,11 +324,11 @@ def test_integration_torch_conversation_blenderbot_400M(self):
             " Hello! How are you doing today? I just got back from a walk with my dog.",
         )
 
-        conversation_1 = Conversation(" Lasagne   hello")
+        conversation_1 = Conversation("Lasagne   hello")
```

Comment:
This needs to be updated because now `_build_conversation_input_ids` does the prefixing automatically.